test/orchard/info_test.clj

```diff
@@ -467,21 +471,49 @@
                :returns int}))
       (is (re-find #"Returns the greater of two" (:doc i))))))
 
+(def some-var nil)
+
+(def workaround
+  "Prevents a clj-kondo warning."
+  replace-first)
+
 (deftest info-undefined-namespace-test
-  (testing "Fully qualified sym can still be resolved"
-    (is (= '{:added "1.2"
-             :ns clojure.string
-             :name upper-case
-             :file "clojure/string.clj"}
-           (select-keys (info/info* {:ns 'gibberish :sym 'clojure.string/upper-case})
-                        [:added :ns :name :file]))))
-  (testing "clojure.core syms can still be resolved"
-    (is (= '{:added "1.0"
-             :ns clojure.core
-             :name merge
-             :file "clojure/core.clj"}
-           (select-keys (info/info* {:ns 'gibberish :sym 'merge})
-                        [:added :ns :name :file])))))
+  (let [current-ns (-> ::_ namespace symbol)]
+    (are [input expected] (= expected
```

Comment:
I don't like `are` as I think it reduces the readability of tests. For me it's better to have more `testing` blocks, as they make it easier to follow what a group of assertions tests exactly.

Reply:
Perhaps we can agree to disagree? I didn't use an `are` of the crazy flavor here. It's a simple mapping of inputs->outputs.

Normally to reach this level of exhaustiveness with manually coded tests, one would have to take 1 to 2 'screenfuls' worth of code. That's generally not too readable - it's so wordy that it's hard to compare the test cases between them.

Normally people give up before even attempting this level of exhaustiveness, so `are` can invite us to do the right thing (focus on the data, compare the tests cases between them, cheaply add more cases if missing)

Reply:
I can live with this, I simply don't like it. Too many years of writing RSpecs I guess. :-) There's also the practical problem that CIDER's test runner doesn't work well with `are`, but that's not a big deal.